app/assets/javascripts/models/beta/story.js

```diff
@@ -136,3 +145,55 @@ export const serialize = (story) => ({
   ...story,
   labels: Label.joinLabels(story.labels)
 });
+
+export const createNewStory = () => ({
+  ...emptyStory,
+  collapsed: false
+});
+
+export const isNew = (story) =>
+  story.id === null;
+
+export const isnotNew = (story) =>
+  story.id !== null;
```

Comment:
Do we really need this method? We can probably use the negation of it in `removeEmptyStory`:

```js
export const removeEmptyStory = (stories) =>
  stories.filter((story) => !isNew(story));
```